docs/before/accounts.md

```diff
@@ -18,31 +17,25 @@ To this end, you will need to follow the steps below (*and for more detailed inf
 4. The experiment spokesperson adds your UNIX account to the [Experiment Manager](https://pswww.slac.stanford.edu/lgbk/lgbk/experiments).
 
 !!! note "Enabling access to the S3DF"
-    The SLAC-wide Shared Science Data Facility ([S3DF](https://lcls-users.readthedocs.io/en/latest/glossary/#s3df)) will soon replace the computer and storage resources managed by LCLS Photon Controls and Data Systems ([PCDS](https://lcls-users.readthedocs.io/en/latest/glossary/#pcds)) currently used for LCLS experiment data management.
+    The SLAC-wide Shared Science Data Facility ([S3DF](https://lcls-users.readthedocs.io/en/latest/glossary/#s3df)) has replaced the computer and storage resources managed by LCLS Photon Controls and Data Systems ([PCDS](https://lcls-users.readthedocs.io/en/latest/glossary/#pcds)) that was  used for LCLS experiment data management prior 2023.
```

Comment:
Minor typo: "used for LCLS experiment data management prior to 2023"

Also maybe we want to say "the **compute** and storage resources..." not computer and storage? Don't have a strong preference either way

Reply:
Thanks, I edited accordingly.